tensorflow/compiler/tf2tensorrt/convert/op_converter.h

```diff
@@ -88,7 +106,8 @@ class OpConverterBase {
 
   // Default allowed data types for the NodeDef data type attribute. The Impl
   // class can override this by implementing the same function.
-  static constexpr std::array<DataType, 2> AllowedDataTypes() {
+  virtual std::vector<DataType> AllowedDataTypes(
+      const OpConverterParams& params) {
```

Comment:
Please add document for the new parameter.

Reply:
DONE